Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -254,9 +252,10 @@ public static void ResetLaunchCount()
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="SystemInformation"/> class.
         /// Initializes static members of the <see cref="SystemInformation"/> class.
         /// </summary>
-        static SystemInformation()
+        public SystemInformation()
```

Comment:
This now allows developers to create more than one instance of this class, and I'm thinking if this would be a problem. I'm not saying we should support multi-threaded scenarios, but we should at least ask developers that use this class to create a singleton of it, by themselves.

Reply:
Is it possible to keep the `_localObjectStorageHelper` static so that information won't be stored multiple times if the user does create more than one?

Reply:
Maybe we just leave the constructor private and implement the Singleton pattern ourselves? There's a discussion about that guidance in general as part of Issue #3134. FYI @Sergio0694

Reply:
If the issue is just with the class itself being `static`, then yeah we could definitely try to just expose a `static SystemInformation Instance { get; }` property from the class. This would also allow users to directly bind to that using `x:Bind` directly from XAML, if they wanted to.

And the constructor could be marked as `private` in that case so that users won't be able to unnecessarily create multiple instances. That `Instance` property is guaranteed to be thread-safe, it's mentioned in that issue that @michael-hawker linked 👍

Also @vgromfeld good find about that .NET Native behavior with static classes, I was not aware the dependency reduction engine wasn't able to work there! TIL 🚀

Reply:
I've updated the code to expose a `static SystemInformation Instance { get; }` property.
The class and its dependencies are properly removed by *.Net Native* when not used. 
We get the same final binary size reduction.